pinot-tools/src/main/java/org/apache/pinot/tools/admin/PinotMinion.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.tools.admin;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+
+/**
+ * Simple shim so that jps -l shows PinotMinion instead of PinotAdministrator
+ */
+public class PinotMinion {
```

Comment:
admin/PinotMinion.java

Reply:
For other components e.g. server. We can starts with both `bin/start-server.sh` and `bin/pinot-admin.sh`. I create this so users can starts with `bin/start-minion.sh` also.

Reply:
Ah i see, that makes sense.